python/xoscar/collective/test/test_pygloo_tcp_store.py

```diff
@@ -0,0 +1,622 @@
+import multiprocessing as mp
+import os
+import platform
+import shutil
+import time
+
+import numpy as np
+
+from ...tests.core import require_unix
+
+fileStore_path = "./collective"
+system_name = platform.system()
+
+
+def worker_allgather(rank):
+    from ..gloo import xoscar_pygloo as xp
+    from ..rendezvous import xoscar_store as xs
+
+    if rank == 0:
+        if os.path.exists(fileStore_path):
+            shutil.rmtree(fileStore_path)
+        os.makedirs(fileStore_path)
+    else:
+        time.sleep(0.5)
+
+    context = xp.rendezvous.Context(rank, 2)
+
+    if system_name == "Linux":
+        attr = xp.transport.tcp.attr("localhost")
+        dev = xp.transport.tcp.CreateDevice(attr)
```

Comment:
Can linux use uv?

Reply:
linux will use tcp, gloo will compile it by setting `USE_LIBUV=OFF`